### ISSUE TITLE
Missing check for NULL

### ISSUE THREAD

**Issue by dkoreshkov** [brg-liuwei/ngx_kafka_module#21]
Hello,
I noticed there's no check for ngx_pcalloc possibly returning NULL at line 466. 
Hope this helps...

**Comment by brg-liuwei** [comment]
Yes, thank you for reading those code, you are pretty right. Would you please send a pull request for this issue?
